app/models/manageiq/providers/ibm_power_vc/cloud_manager/provision_workflow.rb

```diff
@@ -0,0 +1,7 @@
+ManageIQ::Providers::Openstack::CloudManager::ProvisionWorkflow.include(ActsAsStiLeafClass)
+
```

Comment:
Actually just noticed, this isn't an `ActiveRecord::Base` type class (base is `MiqRequestWorkflow`), it is just a normal ruby class so you don't need `ActsAsStiLeafClass`
```suggestion
```

Reply:
addressed in the latest force-push.